kratos/containers/nodal_data.h

```diff
@@ -0,0 +1,226 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
+//
+
+#if !defined(KRATOS_NODAL_DATA_H_INCLUDED )
+#define  KRATOS_NODAL_DATA_H_INCLUDED
+
+// System includes
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "containers/variables_list_data_value_container.h"
+
+namespace Kratos
+{
+///@addtogroup KratosCore
+///@{
+
+///@name Kratos Classes
+///@{
+
+/// Stores all data and dofs which are stored in each elements
+/** This class is the container for nodal data storing:
+ *  Id : The Id of the node 
+*/
+class NodalData
```

Comment:
i think you need to export this class for win compilation. @roigcarlo could u take a brief look to the appveyor errors?